CHANGES.rst

```diff
@@ -17,6 +17,14 @@ Bugfixes
 
 - None so far :)
 
+Improvements
+^^^^^^^^^^^^
+
+- Add "send to speakers" option in Reminders. (:issue:`4305`, :issue:`4958`, :pr:`4966`)
```

Comment:
- We already have an Improvements section above - just add your line there.
- Remove `4305` again - I just noticed that the issue was about something completely different
- You can credit yourself (search for `thanks` in the file to find an example)

Reply:
Damn! Apologies, I don't even know how I missed the `Improvements` section above :facepalm: